server/index.js

```diff
@@ -17,8 +17,31 @@ app.listen(port, () => {
 app.get('/', (req, res) => {
```

Comment:
From **lines 17-19**, are you trying to make the server respond the default GET request to `http://localhost:portNum` with the index.html file? Because if so, then wouldn't you want to have this index.html file in your public/ folder, instead of in the `client/dist/index.html` location?